<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DelayWidget</class>
 <widget class="QWidget" name="DelayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>53</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>40</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>60</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	#this does break dark mode stupid me
	#background-color: white;
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>X.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Delay:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinBox">
     <property name="suffix">
      <string> ms</string>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="clearButton">
     <property name="icon">
      <iconset theme="edit-clear">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="moveUp">
       <property name="icon">
        <iconset theme="go-up">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="moveDown">
       <property name="icon">
        <iconset theme="go-down">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="delButton">
     <property name="icon">
      <iconset theme="list-remove">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>delButton</sender>
   <signal>clicked()</signal>
   <receiver>DelayWidget</receiver>
   <slot>_remove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>707</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>668</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moveDown</sender>
   <signal>clicked()</signal>
   <receiver>DelayWidget</receiver>
   <slot>_moveDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>661</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>629</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moveUp</sender>
   <signal>clicked()</signal>
   <receiver>DelayWidget</receiver>
   <slot>_moveUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>624</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>593</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_remove()</slot>
  <slot>_moveUp()</slot>
  <slot>_moveDown()</slot>
 </slots>
</ui>
